docs/ide/index.rst

```diff
@@ -0,0 +1,25 @@
+.. _ide:
+
+IDE Support
+===========
+
+The following supported Sphinx-Needs language features are intergrated into `Esbonio language server <https://github.com/swyddfa/esbonio>`_.
+
+.. note::
+
+   Currently as long as needs.json exists under {confdir}/_build/needs in your sphinx needs project, all the features are available
+   if your IDE extension or plugin supports `Esbonio language server <https://github.com/swyddfa/esbonio>`_.
+
+
+To use all the Sphinx-Needs language featues,
```

Comment:
Maybe this can be partf of an extra chapter "Usage".

So in the end we would have:
- Installation
- Usage
- Features (maybe this at top as selling argument)